luaui/Widgets/cmd_bar_hotkeys.lua

```diff
@@ -666,7 +669,6 @@ end
 function widget:Initialize()
 	loadEngineBindings('un')
 	reloadBindings()
-
```

Comment:
I liked my newline separating assignment. Not a blocker

Reply:
Accidentally removed while testing. Fixed.